view/frontend/web/js/view/payment/method-renderer/adyen-hpp-method.js

```diff
@@ -199,6 +199,64 @@ define(
 
                         return false;
                     };
+                    // Can be removed after checkout api feature branch goes live since the issuerId key is changed to
+                    // id there and just use the value.issuers in the component
```

Comment:
what do you mean by: checkout api feature branch goes live ?

Reply:
I need this "converter" because this branch was based on the secure-fields-integration feature branch (now rebased to develop) where I still used the directory lookup which returned issuerIds instead of just ids. In the checkout-api feature branch since I will rebase that after we merged this to develop I won't need it anymore since there we are going to work with the checkout api that returns the correct keys.